pages/pathways.jsx

```diff
@@ -0,0 +1,243 @@
+  },
+  {
+    name: "Building a Crowd",
+    type: "Digital Citizenship Pathway",
+    description: 
+      "<p>Here, the theme is mobilizing people to protect the open Internet. We’ll look at strategies for growing our advocacy community, and tools that empower people to speak up.</p>"
+  },
+  {
+    name: "Don’t Feed the Trolls",
+    type: "Digital Citizenship Pathway",
+    description: 
+      "<p>Learn how to protect yourself and your family from cyberbullying and online harassment. We’ll also design tools for better protection.</p>"
+  },
+  {
+    name: "Backdoors and Cryptowars",
+    type: "Digital Citizenship Pathway",
+    description: 
+      "<p>Tech tools can protect our freedom of expression. Play with new features in Firefox 42, learn how to teach Tor, and design the privacy tools we want.</p>"
+  },
+  {
+    name: "What’s Your Policy?",
+    type: "Digital Citizenship Pathway",
+    description: 
+      "<p>Learn about Internet policies worldwide, explore Mozilla’s stance on the issues, and uncover how we can best engage communities around the globe.</p>"
+  },
+  {
+    name: "Voices of Diverse Leaders",
+    type: "MLN Pathway",
+    description: 
+      "<p>This Pathway convenes leaders and educators in order to catalyze new relationships, innovate, and create a more inclusive Web. We’ll create leadership opportunities for participants to empower others to read, write and participate online.</p>",
+    contacts: [
+      {
+        name: "Simona Ramkisson",
+        twitter: "@SimonaRamkisson"
+      },
+      {
+        name: "Julia Vallera",
+        twitter: "@colorwheelz"
+      }
+    ]
+  },
+  {
+    name: "Teaching Web Literacy with Mozilla Clubs",
+    type: "MLN Pathway",
+    description: 
+      "<p>Learn how to teach like Mozilla and facilitate successful events or clubs that are open, participatory, collaborative and engaging.</p>",
+    contacts: [
+      {
+        name: "Amira Dhalla",
+        twitter: "@amirad"
+      },
+      {
+        name: "Andre Garzia",
+        twitter: "@soapdog"
+      }
+    ]
+  },
+  {
+    name: "Learning and Leading",
+    type: "MLN Pathway",
+    description: 
+      "<p>Educators, librarians, and community builders of all experience levels will work toward goals in three interrelated areas: community engagement, technical skills and teach like Mozilla.</p>",
+    contacts: [
+      {
+        name: "Chad Sansing",
+        twitter: "@chadsansing"
+      },
+      {
+        name: "An-Me Chung",
+        twitter: "@anmechung"
+      },
+      {
+        name: "Sam Dyson",
+        twitter: "@samueledyson"
+      }
+    ]
+  }
+];
+
+
+var PathwaysPage = React.createClass({
+  pathways: PathwaysInfo,
+  componentWillMount: function() {
+    this.pathways = this.pathways.map(function(pathway,i) {
+      var whiteBg = (i%2==0) ? "white-background" : "";
+      return (
+        <div className={whiteBg} key={i}>
```

Comment:
don't use `key={i}`, here it'd be something like `key={pathway.name}` or something